pkg/config/config_test.go

```diff
@@ -6,7 +6,9 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
```

Comment:
why added space? the packages are not in the same group?

Reply:
removed